pytest_splinter/hooks.py

```diff
@@ -0,0 +1,6 @@
+import pytest  # NOQA
+
+
+def pytest_splinter_screenshot_dir(config):
```

Comment:
not sure if hooks are meant for this
even your default implementation tell that something is broken: why it's an empty function?

Reply:
new hook is not documented as well

Reply:
Other hook implementations I've seen use an empty function to define the hook. Is this not correct?